station.py

```diff
@@ -129,77 +159,145 @@
 dsn = 0
 checkin_devices = {}
 
+
 def print(*args):
     msg = ""
     for arg in args:
         msg += str(arg) + " "
     logger.warning(msg)
 
+
+# get element from dict by path
+def get_element_by_path(object, keys):
+    if keys:
+        return get_element_by_path(object[keys[0]], keys[1:])
+    return object
+
+
+# generate config object from config mapping and config file
+def config_generator(config_mapping, config_file):
```

Comment:
this should not be needed, the parser should output a sensible config object without any post-processing